frontend/public/components/utils/kebab.tsx

```diff
@@ -109,9 +109,11 @@ export class Kebab extends DropdownMixin {
     return <div id={id}>
       { isDisabled ?
         <Tooltip content="disabled">
-          <div ref={this.dropdownElement} className="co-kebab co-kebab--disabled" >
-            <span className="fa fa-ellipsis-v co-kebab__icon co-kebab__icon--disabled" aria-hidden="true"></span>
-            <span className="sr-only">Actions</span>
+          <div ref={this.dropdownElement} className="co-kebab" >
+            <button type="button" aria-label="Actions" aria-haspopup="true" className="btn btn-link co-kebab__button co-kebab--disabled">
+              <span className="fa fa-ellipsis-v co-kebab__icon co-kebab__icon--disabled" aria-hidden="true"></span>
+              <span className="sr-only">Actions</span>
```

Comment:
This isn't needed if you have `aria-label`

Reply:
removed